tests/demonstrator_scripts/test_dummy_scripts.py

```diff
@@ -16,4 +18,4 @@ def test_dummy_scripts():
 
     E, fc = dummy_paste_strength_stiffness(0, 10)
     assert E.magnitude == pytest.approx(30)
```

Comment:
I guess then accordingly this will change.

Reply:
Changed accordingly